crm_salesperson_planner/models/crm_salesperson_planner_visit_template.py

```diff
@@ -0,0 +1,397 @@
+# Copyright 2021 Sygel - Valentin Vinagre
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+import datetime
+import re
+
+import pytz
+from dateutil import rrule
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class CrmSalespersonPlannerVisitTemplate(models.Model):
+    _name = "crm.salesperson.planner.visit.template"
+    _description = "Crm Salesperson Planner Visit Template"
+
+    name = fields.Char(
+        string="Visit Template Number",
+        required=True,
+        default="/",
+        readonly=True,
+        copy=False,
+    )
+    partner_id = fields.Many2one(
+        comodel_name="res.partner", string="Customer", required=True,
+    )
+    initial_date = fields.Date(
+        string="First Visit", default=fields.Date.context_today, required=True,
+    )
+    next_visit_date = fields.Date(string="Next Visit Date")
```

Comment:
Shouldn't this be computed?

Reply:
Ups, this field should never have existed.